docs/blog/fundamentals-of-structured-data.md

```
@@ -0,0 +1,348 @@
+構造化データとは、ページの情報を表すデータ形式。
+
+> ページに関する情報をさまざまなサイトで活用できるように標準化したデータ形式
+>
+> ref: [Google 検索における構造化データのマークアップの概要](https://developers.google.com/search/docs/appearance/structured-data/intro-structured-data?hl=ja#:~:text=%E3%83%9A%E3%83%BC%E3%82%B8%E3%81%AB%E9%96%A2%E3%81%99%E3%82%8B%E6%83%85%E5%A0%B1%E3%82%92%E3%81%95%E3%81%BE%E3%81%96%E3%81%BE%E3%81%AA%E3%82%B5%E3%82%A4%E3%83%88%E3%81%A7%E6%B4%BB%E7%94%A8%E3%81%A7%E3%81%8D%E3%82%8B%E3%82%88%E3%81%86%E3%81%AB%E6%A8%99%E6%BA%96%E5%8C%96%E3%81%97%E3%81%9F%E3%83%87%E3%83%BC%E3%82%BF%E5%BD%A2%E5%BC%8F)
+
+これによってページに訪れたコンピューターはページのコンテンツ・意図を正確に理解できる。
+
+Google検索の場合、この情報でリッチリザルトを表示している。
+例えばレシピの作成者、材料、カロリー数、加熱時間などの詳細情報を記載すると、各要素がラベル付けされ、それらの情報でも検索できるようになる。
+
+> ![](./assets/fundamentals-of-structured-data/article-rich-result.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)